projects/quickjs/project.yaml

```diff
@@ -0,0 +1,7 @@
+homepage: "https://bellard.org/quickjs/"
+primary_contact: "fabrice@bellard.org"
+auto_ccs :
+- "p.antoine@catenacyber.fr"
+
+sanitizers:
+- address
```

Comment:
any particular reason not to include ubsan? if not, just remove `sanitizers` section and both ASan and UBSan will be used by default

Reply:
I remembered there was a problem with it, but not which one, let's try again with CI